homeassistant/components/ping/device_tracker.py

```diff
@@ -141,14 +148,14 @@ async def async_update(now):
                 )
             )
 
-    async def _async_update_interval(now):
+    async def _async_update_interval(now: datetime) -> None:
         try:
             await async_update(now)
         finally:
             if not hass.is_stopping:
                 async_track_point_in_utc_time(
-                    hass, _async_update_interval, util.dt.utcnow() + interval
+                    hass, _async_update_interval, dt_util.utcnow() + interval
                 )
 
-    await _async_update_interval(None)
+    await _async_update_interval(dt_util.now())
```

Comment:
Why is this line changed?

Reply:
Because the function called has `now` as parameter. 
Could also allow `None` as input but it's not standard so I think looks cleaner.

Reply:
`now` does not seem to be used. I suggest we remove it.

Reply:
```suggestion
    await _async_update_interval()
```